### PR TITLE
Jonmv/zk 3.9.1 os default

### PR DESCRIPTION
**Pull request by jonmv**
@hmusum or @bjorncs please review.

### CODE REVIEW

File: vespa-dependencies-enforcer/allowed-maven-dependencies.txt [hunk 1]
```diff
@@ -128,10 +128,10 @@ org.apache.opennlp:opennlp-tools:${opennlp.vespa.version}
 org.apache.velocity:velocity-engine-core:${velocity.vespa.version}
 org.apache.yetus:audience-annotations:0.12.0
 org.apache.zookeeper:zookeeper-jute:${zookeeper.client.vespa.version}
-org.apache.zookeeper:zookeeper-jute:3.8.1
+org.apache.zookeeper:zookeeper-jute:3.8.0
```
Comment (bjorncs) [inline vespa-dependencies-enforcer/allowed-maven-dependencies.txt:131]: `zookeeper.client.vespa.version` == 3.8.0?

**Review by bjorncs — DISMISSED** [review]

File: vespa-dependencies-enforcer/allowed-maven-dependencies.txt (hunk 1 above)
Comment (jonmv) [reply vespa-dependencies-enforcer/allowed-maven-dependencies.txt:131]: Ah, yes, probably as issue from me mixing up branches a bit.

Comment (jonmv) [reply vespa-dependencies-enforcer/allowed-maven-dependencies.txt:131]: ... or it's the dependency enforcer that doesn't identify the two?

Comment (jonmv) [reply vespa-dependencies-enforcer/allowed-maven-dependencies.txt:131]: (It accepts this list, so I suggest we do too, and it'll be a single version once this version bump is complete.)

**Comment by bjorncs** [comment]
Build fails
```
10:28:42 CMake Error at zookeeper-server/CMakeLists.txt:4 (add_subdirectory):
10:28:42   add_subdirectory given source "zookeeper-server-3.9.1" which is not an
10:28:42   existing directory.
```

File: zookeeper-server/zookeeper-server-3.8.0/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+install_jar(zookeeper-server-3.8.0-jar-with-dependencies.jar)
+# Make symlink so that we have a default version, should be done only in zookeeper-server module
+install_symlink(lib/jars/zookeeper-server-3.8.0-jar-with-dependencies.jar lib/jars/zookeeper-server-jar-with-dependencies.jar)
```
[the remark this answers is not in the document]
Comment (jonmv) [reply zookeeper-server/zookeeper-server-3.8.0/CMakeLists.txt:4]: Ah, thanks, missed this.

**Review by hmusum — APPROVED** [review]

File: zookeeper-server/CMakeLists.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
 # Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 add_subdirectory(zookeeper-server-common)
 add_subdirectory(zookeeper-server)
-add_subdirectory(zookeeper-server-3.9.1)
+add_subdirectory(zookeeper-server-3.8.0)
```
[the remark this answers is not in the document]
Comment (jonmv) [reply zookeeper-server/CMakeLists.txt:4]: The one without numbers now holds 3.9.1.

**Review by bjorncs — APPROVED** [review]

**Pull request merged**